packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -200,6 +202,7 @@ export interface StatefulCallClient extends CallClient {
     createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
     createView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState, options?: CreateViewOptions): Promise<CreateViewResult | undefined>;
     disposeView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState): void;
+    getEnvironmentInfo(): Promise<EnvironmentInfo>;
```

Comment:
This PR isn't currently storing the result of this in the statefulcallclient state so this method seems redundant (because statefulcallclient extends CallClient, a user can just do callClient.feature(Feature.DebugInfo).getEnvironmentInfo)

Reply:
oh never mind I should read the whole PR correctly before commenting lol, deleting this comment